### PR TITLE
DOC-8267 -- Document Release 2.8.2

### PR DESCRIPTION
**Pull request by ibsoln**
https://issues.couchbase.com/browse/DOC-8267

### CODE REVIEW

File: modules/ROOT/pages/_partials/common-releasenotes.adoc [hunk 1]
```diff
@@ -23,6 +23,92 @@ include::{root-partials}block-abstract.adoc[]
...
+// end::other-enhancements-highlights-282[]
+// end::other-enhancements-282[]
+
+
+[#issues-and-resolutions-282]
+=== Issues and Resolutions
+// tag::issue-resolutions-282[]
+
+// tag::issue-resolutions-summary-282[]
+Release 2.8.2 fixes an issue that could lead to a loop when creating mobile tombstone for existing Couchbase Server tombstones.
```
Comment (adamcfraser) [inline modules/ROOT/pages/_partials/common-releasenotes.adoc]:
```suggestion
Release 2.8.2 fixes an issue that could result in a loop when creating a mobile tombstone, if a Couchbase Server tombstone already exists for that key.
```

**Review by adamcfraser — COMMENTED** [review]
Proposed a minor tweak to the description of the issue.

File: modules/ROOT/pages/_partials/common-releasenotes.adoc [hunk 2]
```diff
@@ -23,6 +23,92 @@ include::{root-partials}block-abstract.adoc[]
...
+// end::other-enhancements-highlights-282[]
+// end::other-enhancements-282[]
+
+
+[#issues-and-resolutions-282]
+=== Issues and Resolutions
+// tag::issue-resolutions-282[]
+
+// tag::issue-resolutions-summary-282[]
+Release 2.8.2 fixes an issue that could result in a loop when creating a mobile tombstone, if a Couchbase Server tombstone already exists for that key.
```
Comment (djpongh) [inline modules/ROOT/pages/_partials/common-releasenotes.adoc]: maybe, "...could result in Sync Gateway going into an infinite loop when creating..."

Comment (ibsoln) [reply modules/ROOT/pages/_partials/common-releasenotes.adoc]: addressed https://github.com/couchbase/docs-sync-gateway/pull/431/commits/8c2661e7f7a1bfc73c8589f823688d2873196d76

**Review by amarantha-k — APPROVED** [review]

**Pull request merged**